dictionaries/docker/source/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # syntax=docker/dockerfile:1
-FROM node:14.21.3-alpine
+FROM node:22.9.0-alpine
```

Comment:
I didn't even realize this file was there. I think it was added as a `sample` but incorrectly put in `source`. It isn't used.